slack_rules/slack_ekm_config_changed.py

```diff
@@ -0,0 +1,11 @@
+from panther_base_helpers import slack_alert_context
+
+
+def rule(event):
+    # Only alert on the `ekm_logging_config_set` action
+    return event.get("action") == "ekm_logging_config_set"
+
+
+def alert_context(event):
+    # TODO: Add details to the context
```

Comment:
nit, I suspect this comment can be removed?

Reply:
Once we have this data, this should be updated